File: 프로그래머스/0/120846. 합성수 찾기/합성수 찾기.js
function solution(n) {
    let arr = new Array(n + 1).fill(0);
    
    for (let i = 1; i <= n; i++) {
        for (let j = 1; j <= i; j++) {
            if (i % j === 0) {
                arr[i]++
            };
        }
    }
    return arr.filter((v) => v >= 3).length;
}

File: 프로그래머스/0/120846. 합성수 찾기/합성수 찾기.py
def solution(n):
    arr = [0] * (n + 1)
    
    for i in range(1, n + 1):
        for j in range(1, i + 1):
            if i % j == 0:
                arr[i] += 1
    
    return len([v for v in arr if v >= 3])